src/modules/esl-utils/misc/format.ts

```diff
@@ -67,3 +67,28 @@ export function format(str: string, source: Record<string, any>, matcher: RegExp
     return val === undefined ? match : val;
   });
 }
+
+/** Parses time, passed in seconds or milliseconds
+ * @example
+ * `.3s`, `4.5s`, `1000ms`
+ * @returns milliseconds
+*/
+export function parseTime(timeStr: string): number {
+  const isMilliseconds = timeStr.indexOf('ms') !== -1;
+  if (isMilliseconds) {
+    timeStr = timeStr.replace('ms', '');
+    return parseNumber(timeStr, NaN)!;
```

Comment:
Just `+timeStr` is enough in our situation.
Also, it looks like we need to fix a little bit types for `parseNumber` method

Reply:
There is also a coulee minor points (regarding `parseTime('2wtf') // NaN but not 2000`)